charts/kubefed/charts/controllermanager/templates/clusterrolebindings.yaml

```diff
@@ -27,14 +34,29 @@ subjects:
   name: kubefed-admission-webhook
   namespace: {{ .Release.Namespace }}
 ---
+# This clusterrolebindng grants permissions for the admission webhook to create
+# admission reviews on behalf of the system:anonymous user.
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: anonymous-kubefed-admission-webhook-auth
+# TODO(font) For namespace scoped deployments, create a unique cluster-scoped
```

Comment:
Did you intend to leave this TODO here?  The comment looks the same as the one you've added above.

Reply:
The comment is slightly different as it talks about the lack of support in helm for sharing resources across charts. I left it as a TODO since helm may eventually support it via https://github.com/helm/helm/issues/3189.

Reply:
Hmm, ok.  I was confused because there are large comments with little information as per my comment about duplication, and it wasn't clear to me that this one was different.  Maybe its worth rewording so that it's clear that it is not just boilerplate like the other comments?

Reply:
This one has a TODO so I think it is already different in that regard. But I can change the wording if you feel strongly.